script/script.js

```diff
@@ -3,6 +3,8 @@ const booksAr = new BooksArray();
 const domTb = new DomTable();
 const localSt = new BooksStorage();
 const form1 = document.querySelector('#saveBook');
+const myDate = document.querySelector('#myDate');
+const dt = luxon.DateTime.now().toLocaleString(luxon.DateTime.DATETIME_MED);
 
 function RemoveBooks(idIndex) {
```

Comment:
- [ ] OPTIONAL
It is a nice idea to prefer _arrow functions_ over normal functions, so as to keep your _function_ syntax simpler and cleaner.